geotrek/settings/base.py

```diff
@@ -447,10 +447,24 @@ def api_bbox(bbox, buffer):
 LEAFLET_CONFIG = {
     'SRID': 3857,
     'TILES': [
-        ('OpenTopoMap', 'https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png',
-         {'attribution': 'Données: © Contributeurs OpenStreetMap, SRTM | Affichage: © OpenTopoMap (CC-BY-SA)',
-          'maxZoom': 17}),
-        ('OSM', 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', '© Contributeurs OpenStreetMap'),
+        (
```

Comment:
Is there a reason for changing the order? IMO OpenTopoMap is a relevant default tile layer.